src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyServiceTests.cs

```diff
@@ -1122,28 +1062,29 @@ await using (var context = InMemoryApplicationDbContext(contentDbContextId))
                 Assert.Equal(Immediately, viewModel.PublishingStrategy);
                 Assert.Null(viewModel.ScheduledWithRelease);
                 Assert.Equal(request.Status, viewModel.Status);
-                Assert.Equal(request.Title, viewModel.Title);
+                Assert.Equal("Pupil absence statistics: methodology", viewModel.Title);
```

Comment:
I think we could remove some of the assertions on checking details (title) when updating status and vice versa, checking status when updating title, for brevity.

Reply:
Trimmed down some of the tests now in #2769